http/server_test.go

```diff
@@ -27,6 +28,7 @@ func TestHTTPRelay(t *testing.T) {
 	defer cancel()
 
 	client := drand.NewPublicClient(conn)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Urgh, we need to get rid of these. There should be a mechanism for us to know when the client is ready...

Reply:
yep. agree :-| I'm somewhat unconvinced this fully solves the problem, but i want to see if this stops the occasional 'first group request seen on new server doesn't yet have a group initialized' heisenbug that's popped up on CI a few times.